### PR TITLE
Adds create issue in org repo when bug/feature created in code repo

### PR DESCRIPTION
**Pull request by ashraffouda**
Signed-off-by: Ashraf Fouda <foudaa@codescalers.com>

### CODE REVIEW

File: ays/github_repo/actions.py [hunk 1]
```diff
@@ -690,3 +691,15 @@ def event_new_issue(self, service, event):
...
         args = {'github.repo': service.instance}
         service.aysrepo.new(name='github_issue', instance=str(issue.id), args=args, model=issue.ddict)
+        pattern = '(?P<story_card>\w+)(:)'
+        import re
+        match = re.search(pattern, issue.title)
+        if match:
+            if ("type_bug" in issue.labels or "type_feature" in issue.labels) and repo.type == 'code':
+                repo_name = service.hrd.get("repo.account") + "/" + service.hrd.get("org.repo")
+                org_repo = self.get_github_repo(service, repo_name)
+                org_issue = org_repo.api.create_issue(issue.title)
```
Comment (zaibon) [inline ays/github_repo/actions.py]: shouldn't we also copy the content of the issue and not just the title ?

File: ays/github_repo/schema.hrd [hunk 2]
```diff
@@ -13,4 +13,6 @@ github.client = type:'str' consume:'github_client:1' auto
 
 github.config= type:'str' consume:'github_config:1' auto
 
-repo.emails = type:'str' list
\ No newline at end of file
+repo.emails = type:'str' list
+
+org.repo = type:'str' default:''
```
Comment (zaibon) [inline ays/github_repo/schema.hrd]: maybe consume here is more appropriate then just a string ?

Comment (ashraffouda) [inline ays/github_repo/schema.hrd]: But how can specify the repo with consume? I think it will consume any existing repo

Comment (zaibon) [inline ays/github_repo/schema.hrd]: in the blueprint you specify the instance name of the repo you want to consume.
It's the same as you do now, but the relation has a meaning and you have direct access to the service object.

Comment (ashraffouda) [inline ays/github_repo/schema.hrd]: Ok got it

**Pull request merged**